src/test/java/step_definitions/salesforceDemo.java

```diff
@@ -52,16 +52,16 @@ public void read_the_test_data_from_Excel(String arg1) {
     @When("^navigate to salesforce$")
     public void navigate_to_the_url() throws Exception {
 
-//        KeywordUtil.navigateToUrl("https://testing-f5-dev-ed.develop.my.salesforce.com");
-        dataMap = ExcelDataUtil.getTestDataWithTestCaseID("Salesforce", "TestData1");
-        KeywordUtil.navigateToUrl(dataMap.get("URL"));
+        KeywordUtil.navigateToUrl("https://testing-f5-dev-ed.develop.my.salesforce.com");
+//        dataMap = ExcelDataUtil.getTestDataWithTestCaseID("Salesforce", "TestData1");
```

Comment:
Could you please remove these commented lines as well if not required?

Reply:
These are required for now until we find solution for different URLs problem